subsys/net/l2/canbus/canbus.c

```diff
@@ -1,49 +1,1333 @@
 /*
- * Copyright (c) 2019 Intel Corporation.
+ * Copyright (c) 2019 Alexander Wachter.
```

Comment:
No, let the original ©, just add yours below.

Reply:
> No, let the original ©, just add yours below.

Umm, no, separate renaming of the old CANBUS to CANBUS_RAW vs creation of new CANBUS into 2 separate commits, so it was clear what happens here. And perhaps, it should be canbus/6locan.c after all...

Reply:
@tbursztyka This is because I changed the file-names. The canbus.c is (c) by, but the canbus_raw.c is (c) by Intel. I'll change it to 6locan as @pfalcon suggested.

Reply:
ok missed that